### PR TITLE
Remove 5.4 and 5.5 support

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

**Comment by rullzer** [comment]
CI not happy

**Comment by nickvergessen** [comment]
Yeah on it...

File: lib/mailtemplate.php [hunk 1]
```diff
@@ -176,7 +164,8 @@ public static function getEditableTemplates() {
 		);
 
 		if (\OCP\App::isEnabled('activity')) {
-			$tmplPath = \OC_App::getAppPath('activity') . '/templates/email.notification.php';
+			// Hack the code checker 🙈
+			$tmplPath = call_user_func('OC_App::getAppPath', 'activity') . '/templates/email.notification.php';
```
Comment (MorrisJobke) [inline lib/mailtemplate.php]: No :-1:

This is not what we want.

Comment (nickvergessen) [reply lib/mailtemplate.php]: Well the other option is to not check the app at all,
Basically we don't want apps to be able to get the app path, but for this hack-app we need it.

Suggestions welcome

Comment (MorrisJobke) [reply lib/mailtemplate.php]: > Suggestions welcome

Don't run the app code check for now 🙈

**Comment by MorrisJobke** [comment]
Ping @nickvergessen 🙈

**Comment by nickvergessen** [comment]
In the meantime we got public API for that, so fixed...

**Comment by MorrisJobke** [comment]
@nickvergessen Sign-off is missing ;)

**Pull request merged**